packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -97,6 +97,15 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
     // If there is no match, that means that the time was past the last data point.
     // In this case, the best match for this price is the current price.
     if (match === undefined) {
+      let returnPrice = this.invertPrice ? this._invertPriceSafely(this.currentPrice) : this.currentPrice;
+      if (verbose) {
+        console.group(`\n(${this.exchange}:${this.pair}) No OHLC available @ ${time}`);
+        console.log(`- ✅ Falling back to Current Price:${this.web3.utils.fromWei(returnPrice.toString())}`);
```

Comment:
This phrasing of falling back to current price is a bit weird to me. It seems like its just using current price because the time is > earliest historical time. But this makes me think its some kind of warning or error. WDYT?

Reply:
Great observation, it's more accurate to say something like "Time is later than earliest historical time, fetching current price"

Reply:
yea i think that would be better, less alarming